pkg/daemon/sync.go

```diff
@@ -53,6 +54,7 @@ func (d *Daemon) Sync(ctx context.Context, started time.Time, newRevision string
 	if d.GitSecretEnabled {
 		ctxt, cancel := context.WithTimeout(ctx, d.GitTimeout)
 		if err := working.SecretUnseal(ctxt); err != nil {
+			updateSyncManifestsMetric(0, 1)
```

Comment:
Remove these ones -- the metric measures the number of manifests (/resources), so setting 0 or 1 is the wrong unit. You will be able to see the success or failure of syncs in the metric `flux_daemon_sync_duration`.

Reply:
Fixed.